docs/content/guides/developer/getting-started/alternative-faucets.mdx

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I think we can delete this one. Look at the faucet topic and let me know what you think. It has the recommended web app and then puts these in a tabbed layout beneath that.